README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/SumoLogic/sumologic-kubernetes-collection.svg?branch=main)](https://travis-ci.org/SumoLogic/sumologic-kubernetes-collection) [![Docker Pulls](https://img.shields.io/docker/pulls/sumologic/kubernetes-fluentd.svg)](https://hub.docker.com/r/sumologic/kubernetes-fluentd)
+[![Docker Pulls](https://img.shields.io/docker/pulls/sumologic/kubernetes-fluentd.svg)](https://hub.docker.com/r/sumologic/kubernetes-fluentd)
```

Comment:
Why not add GitHub Actions badge in place of Travis badge?

Example: https://github.com/jedr/dude-wheres-my-stocks/blame/main/README.md#L3

Reply:
Let's do that once we have actions stabilised.